app/schema.graphql

```diff
@@ -0,0 +1,103 @@
+scalar Datetime
+
+type Author {
+  name: String
+  email: String
+}
+
+# git commit data.
+type Commit {
+  sha1: String!
+  author: Author!
+  date: Datetime!
+  message: String!
+  refs: [Ref]
+  objectName: ObjectName!
+}
+
+# file info and metadata.
+type Info {
+  # base name of the file.
+  name: String!
+  # length in bytes.
+  size: Int!
+  # if it's a directory.
+  isdirectory: Boolean!
+  # any flags git stores, currently only executable.
+  flags: [Int]!
+  # last commit which affected this path.
+  lastcommit: Commit!
```

Comment:
This will be more interesting. Not sure how to model this currently, but if you could make an issue I'll start thinking about this as a sole feature. At the moment we could hardcode it as the head commit of the `History` :grimacing:

Reply:
radicle-dev/radicle-surf#11